lib/soapy_cake/admin.rb

```diff
@@ -102,6 +102,10 @@ def affiliate_signup(opts = {})
       run Request.new(:admin, :signup, :affiliate, opts)
     end
 
+    def media_types(*)
```

Comment:
No we don't. It's a leftover of the testing logic that passes a default argument of `{}` to all methods. I will fix that.